### PR TITLE
feat: Show warning when starting / joining a call without microphone

### PR DESCRIPTION
**Pull request by bennyn**
Audio input error view model can be tested calling:

```js
wire.app.repository.calling.showNoAudioInputModal()
```

Preferences navigation can be tested opening:

```
url/#/preferences/av
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,22 +123,34 @@ All Wire translations are crowdsourced via [Crowdin](https://crowdin.com/project
...
+
+```yaml
+api_key: your-account-api-key
+api_token: your-personal-access-token
+project_identifier: wire-webapp
+username: your-username
+```
+
+**Usage:**
+
```
Comment (ffflorian) [inline README.md:143]: I think this should be done in another PR.

**Review by ffflorian — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bennycode) [reply README.md:143]: Is that the only change that you request?

Comment (ffflorian) [reply README.md:143]: I think so, yes

**Review by ffflorian — APPROVED** [review]
